Automattic/vip-workflow-plugin
# Add date time support to Editorial Metadata

## Description

- Add support for date time as an editorial metadata field. This mirrors the way gutenberg shows the publish time of a post, which allows it to not clutter up the sidebar.
- Change the fields offered out of box, to be unique to our plugin - Embargo Date, Expiry Date and Legal Review. 
- Notification test that tested the cron code was flaky so I removed it. Need to re-think what that would look like

![Screenshot 2024-09-20 at 3 35 16 pm](https://github.com/user-attachments/assets/549e6761-9502-4e83-8abe-a59315759725)

## Steps to Test

- Delete the editorial metadata terms that are on your local env
- Delete the `vw_editorial_metadata_options` option from your local env
- Ensure the new ones are created from scratch
- Test out the new date time picker
